src/components/FindPark.vue

```diff
@@ -1,23 +1,31 @@
 <template>
   <section>
     <h2>Let's Go Play!</h2>
-    <button @click="searchByLocation">Find a dog park near me!</button>
+    <button
+      :class="{ disabled: !this.$store.state.geolocation }"
```

Comment:
This is awesome! `:class` seems really flexible. I'll definitely be using this on my branch.